### PR TITLE
feature(condo): DOMA-5272 extend meter schema to support integration

### PR DESCRIPTION
**Pull request by dkoviazin**
Added importId + importRemoteSystem to identify meter from integration
Added meta field to store additional settings from integration

### CODE REVIEW

**Comment by pahaz** [comment]
Should we remove BillingMeter ?

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/meter/schema/Meter.js [hunk 1]
```diff
@@ -163,6 +164,21 @@ const Meter = new GQLListSchema('Meter', {
             isRequired: true,
             defaultValue: false,
         },
+        meta: {
+            schemaDoc: 'Meter metadata. Can be used to store additional settings from integrations',
```
Comment (SavelevMatthew) [inline apps/condo/domains/meter/schema/Meter.js:168]:
```suggestion
            schemaDoc: 'Meter metadata. Can be used to store additional settings from external sources, such as billing integrations or miniapps',
```

File: apps/condo/domains/meter/schema/Meter.js [hunk 2]
```diff
@@ -163,6 +164,21 @@ const Meter = new GQLListSchema('Meter', {
...
+            type: Json,
+            isRequired: false,
+        },
+        importRemoteSystem: {
+            schemaDoc: 'External provider for meter',
+            type: Text,
+            kmigratorOptions: { null: true, unique: false },
+        },
+        importId: {
```
Comment (SavelevMatthew) [inline apps/condo/domains/meter/schema/Meter.js:177]: Store it in meta, that's the purpose of this field

File: apps/condo/domains/meter/schema/Meter.js [hunk 3]
```diff
@@ -163,6 +164,21 @@ const Meter = new GQLListSchema('Meter', {
             isRequired: true,
             defaultValue: false,
         },
+        meta: {
+            schemaDoc: 'Meter metadata. Can be used to store additional settings from integrations',
+            type: Json,
+            isRequired: false,
+        },
+        importRemoteSystem: {
```
Comment (SavelevMatthew) [inline apps/condo/domains/meter/schema/Meter.js:172]: Why we need it here? We have `MeterReadingSource` schema to clarify, where readings are came from...

**Review by SavelevMatthew — CHANGES_REQUESTED** [review]
Cannot understand what's importId is doing outside of meta, which is the place for storing external information for apps / integrations / data providers and so on (according to our conventions)

It's also not clear to me, how our contractors was able to sync all their data without this changes...And then billings are creating specific fields there for your own needs. IMO they should have unified flow...Since we treat billings integrations same as miniapps

**Review by SavelevMatthew — APPROVED** [review]

**Pull request merged**